plugin/core/url.py

```diff
@@ -60,6 +60,9 @@ def parse_uri(uri: str) -> Tuple[str, str]:
             else:
                 return parsed.scheme, path
         return parsed.scheme, path
+    elif parsed.scheme == '' and ':' in parsed.path:
```

Comment:
I'm not sure though whether this condition is really correct. It seems that colons are allowed in the actual path too.

Reply:
I think it should be fixed now.